code/mob/living/life/bodytemp.dm

```diff
@@ -48,7 +48,7 @@
 			return ..()
 
 		//shivering status chance
-		if (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp && !owner.is_cold_resistant())
+		if (owner.bodytemperature + owner.temp_tolerance < owner.base_body_temp && !owner.is_cold_resistant() && isalive(owner))
```

Comment:
amb, grouping, parentheses please

Reply:
it's just `&&` and basic operators, I think it's fairly unambiguous.

Reply:
specifically the addition and comparison

Reply:
it's even more ambiguous now